src/checkstatus/SSMAutomationExecutionStatusWaiter.py

```diff
@@ -0,0 +1,31 @@
+import botocore
+import constants
+import waiter_acceptor_config
+
+def check_automation_execution_status(automation_execution_id, max_attempts, ssm):
```

Comment:
do we see the need to validate the automation_execution_id

Reply:
no - automation id is passed from the context of the previous execution